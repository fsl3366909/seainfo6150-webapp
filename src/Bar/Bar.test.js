import React from "react";
import Bar from "./Bar";

describe("Bar tests", () => {
  it("renders correctly", () => {
    const { container } = render(<Bar categoryId={0} productId="baz" />);
    expect(container).toMatchSnapshot();
  });
});
