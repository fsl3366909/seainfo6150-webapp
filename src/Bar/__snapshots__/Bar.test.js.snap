// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bar tests renders correctly 1`] = `
<div>
  <div>
    The bar page with categoryId of 
    0
     and productId of 
    baz
  </div>
</div>
`;
